Google.Api.Gax.Grpc/Rest/GrpcRestAdapter.cs

```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * Copyright 2020 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using Google.Protobuf.Reflection;
+using Grpc.Core;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Google.Api.Gax.Grpc.Rest
+{
+    /// <summary>
+    /// Implementation of <see cref="GrpcAdapter"/> that uses HTTP/1.1 and JSON,
+    /// but via a gRPC <see cref="CallInvoker"/>.
+    /// </summary>
+    public sealed class RestGrpcAdapter : GrpcAdapter
```

Comment:
The file is named `GrpcRestAdapter`, whereas this class is named `RestGrpcAdapter`; I assume that at least one of these should be changed ;)